server/serverRun.py

```diff
@@ -181,6 +190,24 @@ def all_users():
     return render_page('user.html', users=users)
 
 
+@app.route('/items', methods=['GET'])
+def item():
```

Comment:
Match the name pluralness
```suggestion
def items():
```